app/src/main/java/org/mozilla/fenix/tabstray/browser/BrowserTabsAdapter.kt

```diff
@@ -71,7 +71,9 @@ class BrowserTabsAdapter(
 
         holder.tab?.let { tab ->
             holder.itemView.mozac_browser_tabstray_close.setOnClickListener {
-                interactor.close(tab)
+                if(!holder.itemView.mozac_browser_tabstray_close.isGone) {
```

Comment:
Do we need this extra check if the view is already gone?

Reply:
Yes, when it came to going from "isInvisible" to "isGone" in showTabIsMultiSelectEnabled, I still found that the bug was still there even though it was considered GONE. This extra check was added in order to make sure that the click button was not active when it's set to GONE.

Reply:
Then I'd say the problem is somewhere else and this issue just got a bit more tricky.
To note:
- there is no issue for the close button of the list item. Same kotlin code that changes visibility, different xml structure.
- just for testing, if we modify `showTabIsMultiSelectEnabled` to be
```
    fun showTabIsMultiSelectEnabled(isSelected: Boolean) {
        itemView.selected_mask.isVisible = isSelected
        closeView.isGone = true
    }
```
the issue doesn't reproduce even for the grid item.
So `View.GONE` should work but needs more investigation as to why it currently doesn't.
And why just for the grid item.
There could be more issues because of that same underlying cause or that same problem could exist in other parts of the app.

Reply:
good point, I think in my previous tests I did check to see if `closeView.isGone = true` would result in the view disappearing and it did. 

Is their possibly a starting point that we could pin point where the problem could occur? It's interesting how list side is practically the same except for XML. I'll look into it more until then!